plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemoryConnectorTest.java

```diff
@@ -117,17 +117,6 @@ protected TestTable createTableWithDefaultColumns()
         throw new SkipException("Cassandra connector does not support column default values");
```

Comment:
Can we fix this line in a new commit or PR? > "Cassandra"

Reply:
Good catch. Adding another commit.